app/assets/stylesheets/application.css

```diff
@@ -2,15 +2,185 @@
  * This is a manifest file that'll be compiled into application.css, which will include all the files
  * listed below.
  *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
  * vendor/assets/stylesheets directory can be referenced here using a relative path.
  *
  * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
  *= require_tree .
  *= require_self
- *= require font-awesome
  */
+
+body {
+  margin: 20px 40px;
+}
+
+.user-data {
+  display: flex;
+  flex-direction: row;
+  gap: 40px;
+  margin-bottom: 20px;
+}
```

Comment:
@BenMukebo Thanks for your comment. I have question how I can compare login page to signup page.  I think you are putting two different comparisons.
Am I am going to maintain the CSS?

Reply:
Hello @habasm, I've used the login page because we did not provide the sign_up page layout. But I think you should pay closer attention to the project requiement 👇🏻 

![image](https://github.com/habasm/BlogApp/assets/79058364/ccbbc4c2-4747-44aa-8b77-89748bdddb8b)